compact/tof_endcap.xml

```diff
@@ -51,8 +51,8 @@
     <constant name="ForwardTOFLayer1_zmin"      value="ForwardTOFMod1_zmin - TOFEndcapLayer_thickness / 2" />
     <constant name="ForwardTOFLayer2_zmin"      value="ForwardTOFMod2_zmin - TOFEndcapLayer_thickness / 2" />
 
-    <constant name="BackwardTOFMod_rmin"        value="BackwardPIDRegion_tan * BackwardTOF_zmin" />
-    <constant name="BackwardTOFMod_rmax"        value="BackwardPIDRegion_rmax" />
+    <constant name="BackwardTOFMod_rmin"        value="12*cm" />
+    <constant name="BackwardTOFMod_rmax"        value="64*cm" />
```

Comment:
Please do not change parametrization with hard-coded values.

Reply:
Done